apps/opik-frontend/src/components/layout/SideBar/SideBar.tsx

```diff
@@ -216,7 +216,6 @@ const SideBar: React.FunctionComponent<SideBarProps> = ({
 
   const { data: promptsData } = usePromptsList(
     {
-      workspaceName,
```

Comment:
Sorry for the misunderstanding here but you need the `workspaceName` as parameter, what you don't need is to send it in the request (it's automatically sent in the request header)

You have to pass it to the query because it should be part of the query key (otherwise on changing workspace name it'll display the cache from previous workspace)